src/views/v2/tickets/item/quantity/add.php

```diff
@@ -14,8 +14,8 @@
  *
  * @version TBD
  *
- * @var $this Tribe__Tickets__Editor__Template
- * @var Tribe__Tickets__Ticket_Object $ticket The ticket object.
+ * @var	Tribe__Tickets__Editor__Template	$this Template object.
+ * @var	Tribe__Tickets__Ticket_Object		$ticket The ticket object.
```

Comment:
* :no_entry_sign: Spaces must be used for mid-line alignment; tabs are not allowed
* :no_entry_sign: Spaces must be used for mid-line alignment; tabs are not allowed

<sub>hash: abd5ca71c7dca75cecc63018703e2b39</sub>

Reply:
```suggestion
 * @var	Tribe__Tickets__Ticket_Object    $ticket The ticket object.
```